src/messages.rs

```diff
@@ -194,6 +194,11 @@ pub struct GroupList {
 }
 
 impl GroupList {
+    /// Gets the `route`-th name in the group list.
+    pub fn get_routeth_name(&self, dst_name: &XorName, route: usize) -> &XorName {
+        RoutingTable::get_routeth_name(self.pub_ids.iter().map(|id| id.name()), dst_name, route)
+    }
```

Comment:
Ah, right. Or `Borrow<XorName>` or something. :confused: 
Let's keep it as it is for now then.